epp-client/src/tests/int.rs

```diff
@@ -0,0 +1,30 @@
+use super::CLTRID;
```

Comment:
Is there a reason for not putting this outside `src`, in the `tests` directory?

Reply:
Yeah, it belongs in `tests/`. I'll move it.